### DISCUSSION TITLE
Solution for Ch2 so far

### DISCUSSION THREAD

**Discussion by lsg6140** [jjdosa/sosu#16] [General]
[Ch2.pdf](https://github.com/jjdosa/sosu/files/9892712/Ch2.pdf)

**Comment by leejiwon8433** [comment]
Please review my exercises
[Exercise 2.3.3, 2.3.4.pdf](https://github.com/jjdosa/sosu/files/9990816/Exercise.2.3.3.2.3.4.pdf)

**Reply by jjdosa** [reply]
Thank you for sharing the solutions. They look great to me!
